src/notation/internal/notationinteraction.cpp

```diff
@@ -1570,7 +1570,7 @@ bool NotationInteraction::doDropStandard()
 //! NOTE: Helper method for NotationInteraction::drop. Handles drop logic for text base items & symbols (returns "accepted")
 bool NotationInteraction::doDropTextBaseAndSymbols(const PointF& pos, bool applyUserOffset)
 {
-    EngravingItem* el = elementAt(pos);
+    EngravingItem* el = m_dropData.dropTarget ? const_cast<EngravingItem*>(m_dropData.dropTarget) : elementAt(pos);
```

Comment:
important when dragging symbols from a palette - if a drop target had already been visually identified, it should use that, rather than trying find a new one. And works correctly for other element types too, in fact I never observed a situation where elementAt( ) got called.